src/lib/assets/logos/FrequencyIcon.svelte

```diff
@@ -1,9 +1,18 @@
 <script lang="ts">
   import type { SVGAttributes } from 'svelte/elements';
 
-  let { ...rest }: SVGAttributes<SVGElement> = $props();
+  interface Props extends SVGAttributes<SVGSVGElement> {
+    fillColor?: string;
+    letterColor?: string;
```

Comment:
@claireclark1 Not sure what you mean by adding handling for the other colors. You set the color here, I just have a default set so it can show up in storybook.

Reply:
Since it is always a two color icon, I cannot use `currentColor` like we do on the logo.